.github/PULL_REQUEST_TEMPLATE/StandardPR.md

```diff
@@ -0,0 +1,32 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce to this project?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] Lint and front-end tests pass locally with my changes
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have added necessary documentation (if appropriate)
+
+## What you learned
+
+If there is anything you learned that may be helpful to other developers, please mention it here.
+
+## Further comments
+
+If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...
+
+### Reviewers: 
+
```

Comment:
just wondering if this section is meant to list the reviewers or a place for reviewers to provide their feedback or both?

Reply:
Ah yeah I believe it's to list the people that reviewed it? I'm not totally sure though, I'm pulling this from a template. I'd assume the best place for feedback to go is in comments.

Reply:
ok, yeah, i agree with you.